llvm/lib/Target/AMDGPU/VOP1Instructions.td

```diff
@@ -1471,17 +1471,34 @@ def : GCNPat<
 >;
 
 def : GCNPat<
-  (i64 (anyext i16:$src)),
+  (i64 (UniformUnaryFrag<anyext> i16:$src)),
+  (REG_SEQUENCE VReg_64,
+     (i32 (COPY $src)), sub0,
+     (V_MOV_B32_e32 (i32 0)), sub1)
```

Comment:
Why VReg_64 and V_MOV_B32? I would expect SReg_64 and S_MOV_B32.

Stepping back a bit, I do not understand exactly how these patterns help. `UniformUnaryFrag` is not enough to ensure that the input is in an SGPR, since sometimes you get uniform values in VGPRs.

Reply:
It seems using SReg_64 here introduces some related changes in the test. I would rather not do it in this patch